vignettes/fkClass.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "fkClass"
+author: "Clair Blacketer, Katy Sadowski"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    number_sections: yes
+    toc: yes
+---
+
+## fkClass
+
+**Check Name**: fkClass
+<br>**Level**: Field check
+<br>**Context**: Verification
+<br>**Category**: Conformance
+<br>**Subcategory**: Computational
+<br>**Severity**: CDM convention &#x26A0;
+
+**Description**: The number and percent of records that have a value in the **cdmFieldName** field in the **cdmTableName** table that do not conform to the **fkClass** class. 
+
+**Definition**: There is the occasional field in the OMOP CDM that expects not only concepts of a certain domain, but of a certain concept class as well. The best example is the `drug_concept_id` field in the `DRUG_ERA` table. Drug eras represent the span of time a person was exposed to a particular drug *ingredient*, so all concepts in `DRUG_ERA.drug_concept_id` must be of the drug domain and ingredient class.
+
+- *Numerator*: The number of rows in the table where the standard concept ID field contains a concept that does not conform to the specified concept_class_id. This numerator specifically excludes concept_id 0 
+- *Denominator*: The total number of rows in the specified cdm table. This denominator includes rows with concept_id 0
+- *Related CDM Convention(s)*: This check is specific to [DRUG_ERA](http://ohdsi.github.io/CommonDataModel/cdm54.html#DRUG_ERA) and [DOSE_ERA](http://ohdsi.github.io/CommonDataModel/cdm54.html#DOSE_ERA) as the `drug_concept_id`s in these tables must be ingredients, which are denoted by the concept class ‘ingredient’
+- *CDM Fields/Tables*: This check is designed to be run on the `drug_concept_id` field in the DRUG_ERA and DOSE_ERA tables 
+- *Default Threshold Value*: 0
+
+**User Guidance**: This check identifies whether records with the correct concepts were written to the correct tables as derived from drug_exposure. If incorrect concepts are allowed to persist, a study package could run on the DRUG_ERA and DOSE_ERA tables but may not produce expected results. 
+
+You may inspect the violating rows using the following query:
+
+```
+-- @cdmTableName.@cdmFieldName is either drug_era.drug_concept_id or dose_era.drug_concept_id
+
+SELECT 
+  '@cdmTableName.@cdmFieldName' AS violating_field, 
+  co.concept_class_id AS violating_class,
+	cdmTable.* 
+FROM @cdmDatabaseSchema.@cdmTableName cdmTable
+	LEFT JOIN @vocabDatabaseSchema.concept co ON cdmTable.@cdmFieldName = co.concept_id
+WHERE co.concept_id != 0 
+  AND (co.concept_class_id != 'ingredient') 
+```
+
+*ETL Developers*\
```

Comment:
I propose to make these itelic words into h3 headings (i.e. `### ETL Developers`)